src/CloudService/custom/New-AzCloudService.ps1

```diff
@@ -0,0 +1,646 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+<#
+.Synopsis
+Create a CloudService Resource
+.Description
+Create a CloudService Resource 
+.Link
+https://docs.microsoft.com/powershell/module/az.cloudservice/new-azcloudservice
+#>
+
+function New-AzCloudService {
+    [OutputType('Microsoft.Azure.PowerShell.Cmdlets.CloudService.Models.Api20210301.ICloudService')]
```

Comment:
Hey, I think there is a better way to implement New-AzCloudService rather than handcraft whole cmdlet.
- First, change the branch to aa62339078c075d81aa1019c62450805f750e2d7
![image](https://user-images.githubusercontent.com/19869716/124071047-607e9480-da71-11eb-8f67-dfd418920931.png)
![image](https://user-images.githubusercontent.com/19869716/124071089-6ffddd80-da71-11eb-84ca-1d2b2710ffb8.png)
- Re-generate a cmdlet then you can find a raw generated  New-AzCloudService
- Adapt generated cmdlet by customization.

Reply:
so in this PR I am adding two new parameter sets to New-AzCloudService. The default parameter set is generated from Swagger, but the two new parameter sets have a lot more functionality than what we could generate from swagger. it takes in cloudService configurations from CsCfg, CsDef, and CsPkg files and validates that all the configuration is sound. It also creates other resources if they're not already created, which before it required them to have those resources created. 

pretty much, we only added here to customization what could not be generated from swagger.